internal/registry/hub/docker_hub.go

```diff
@@ -81,25 +85,75 @@ func configFromSecret(secret *v1.Secret) (string, string, error) {
 	return stanza.Username, stanza.Password, nil
 }
 
+type config struct {
+	ContainerConfig struct {
+		Labels map[string]string
+	} `json:"container_config"`
```

Comment:
Any specific reason this is not a type on it's own?

I think I'd prefer to see this defined outside of the `config` and then this being embedded instead of defined within.

Reply:
no reason! i'll update.